app/payment/session.go

```diff
@@ -76,6 +90,30 @@ func GetPayChs(s perun.SessionAPI) []PayChInfo {
 	return payChInfos
 }
 
+// SubPayChProposals registers a subscription for payment channel proposals.
```

Comment:
```suggestion
// SubPayChProposals subscribe for payment channel proposals.
```

Reply:
Changed to sets up a subscription for ...